lib/Interpreter/Interpreter.cpp

```diff
@@ -163,8 +171,15 @@ namespace cling {
     m_RawInputEnabled(false) {
 
     m_LLVMContext.reset(new llvm::LLVMContext);
+    if (!m_LLVMContext)
+      return;
     m_DyLibManager.reset(new DynamicLibraryManager(getOptions()));
+    if (!m_DyLibManager)
+      return;
+
     m_IncrParser.reset(new IncrementalParser(this, llvmdir));
+    if (!m_IncrParser || !m_IncrParser->isValid(false))
+      return;
```

Comment:
I am not sure I understand 'for output to work'. Is there a particular use case for this?